apps/SageAccountWeb/src/components/SageResourcesPage.tsx

```diff
@@ -22,9 +22,6 @@ export const SageResourcesPage = (props: SageResourcesPageProps) => {
         sx={{
           margin: '0 auto',
           width: '900px',
-          backgroundColor: '#FFF',
-          boxShadow: '0px 2px 6px rgba(53, 58, 63, 0.1)',
-          borderRadius: '5px',
```

Comment:
Use overridden Paper theme to get box-shadow and border-radius - no visible changes to styling

main | feature
:---:|:---:
![main_sageresources](https://user-images.githubusercontent.com/26949006/229908094-cf5c22c5-8220-43c8-bf50-4e52f05ac595.png) | ![feature_sageresources](https://user-images.githubusercontent.com/26949006/229908080-ed87a7b0-a9f9-48e5-927a-33de9a04bac9.png)